Debian rust source does not build for windows anyway

and we cannot link against the Windows cng interface, at any rate.

Index: sequoia-openpgp-mt/Cargo.toml
===================================================================
--- sequoia-openpgp-mt.orig/Cargo.toml
+++ sequoia-openpgp-mt/Cargo.toml
@@ -56,14 +56,6 @@ features = [
 ]
 default-features = false
 
-[target."cfg(windows)".dev-dependencies.sequoia-openpgp]
-version = "2"
-features = [
-    "crypto-cng",
-    "__implicit-crypto-backend-for-tests",
-]
-default-features = false
-
 [badges.gitlab]
 repository = "sequoia-pgp/sequoia-openpgp-mt"
 
